pnpm-lock.yaml

```diff
@@ -17,7 +17,7 @@ specifiers:
   '@nuxtjs/style-resources': ^1.0.0
   '@nuxtjs/svg': ^0.3.0
   '@playwright/test': ^1.16.2
-  '@testing-library/jest-dom': ^5.14.1
+  '@testing-library/jest-dom': ^5.16.1
```

Comment:
When I ran `pnpm install` locally, there were `pnpm-lock.yaml` changes for `lodash.throttle`.